### ISSUE TITLE
Multiple URL in per entry

### ISSUE THREAD

**Issue by LatinSuD** [pfn/keepasshttp#400]
It would be nice if you could have several URL per keepass entry.

For example, the WeAutoType plugin supports a syntax for that. With this syntax you only need one Keepass entry, and in the "Auto-Type" tab put additional URL in a special format similar to this:

??:URL:https://citi.com
??:URL:https://citibank.com

And so on.

There was a similar request #107, but the solution given has some drawbacks. I don't want my file full with lots of duplicate entries.

**Comment by pseudocoder** [comment]
I agree - the duplicate entry approach in KeePass isn't very user friendly for either entry or viewing a database.

The above suggestion would be much cleaner, and I'd suggest making it flexible by allowing a regex to specify which fields should be searched, with a default value of ^URL (== ^URL.*$) or even just URL (== ^.*URL.*$)

**Comment by pseudocoder** [comment]
Or even easier - allow URLs in the Notes field separated by newline, very user friendly for viewing and editing.
